modules/sno-du-disabling-ntp.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-disabling-ntp_{context}"]
+= Disabling Network Time Protocol (NTP)
+
+After the system is configured for Precision Time Protocol (PTP), you need to
+remove NTP from impacting the system clock.
+
+.Procedure
+
+* No configuration changes are needed. Use the provided settings:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: disable-chronyd
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+        systemd:
```

Comment:
If this is needed, then the systemd section should be under spec.config instead of under spec.config.ignition.version.

Reply:
moved this section.